zh/textsearch/pods/ranker/ranker.yml

```diff
@@ -0,0 +1,10 @@
+!BM25Ranker
```

Comment:
For `0.8.0`, you will add one line at https://github.com/jina-ai/jina-hub/blob/master/rankers/BM25Ranker/__init__.py#L19

```
required_keys = {'text', 'length'}
```